com/metacube/SCF/Assignment2/HexCalc.java

```diff
@@ -0,0 +1,154 @@
+package GET2018.com.metacube.SCF.Assignment2;
+
+public class HexCalc {
+	
+	private int decValue1, decValue2, answer;
+	private String digits  = "0123456789ABCDEF";
+	private String result;
+
```

Comment:
Class level variables are needed only when they are to be initialized from constructor. In your case we can simply create local variables. So avoid using class level variables until unless really necessary.